accord-core/src/main/java/accord/coordinate/ReadCoordinator.java

```diff
@@ -251,7 +251,7 @@ protected void start(Set<Id> to)
 
     public void start()
     {
-        Set<Id> contact = newHashSetWithExpectedSize(maxShardsPerEpoch());
+        Set<Id> contact = newLinkedHashSetWithExpectedSize(maxShardsPerEpoch());
```

Comment:
I think it is a vestigial requirement that this be a set. We can just allocate a `List`, and modify the `start` method to accept a `List`.